src/clip.sol

```diff
@@ -362,6 +367,7 @@ contract Clipper {
         dog.digs(ilk, sales[id].tab);
         vat.flux(ilk, address(this), msg.sender, sales[id].lot);
         _remove(id);
+        // TODO: add id?
```

Comment:
I think this is a yes. Event without the parameter doesn't seem useful.

Reply:
Yeah, it should have the `id`.